### PR TITLE
fix: added minor fixes to api re-deployment

### PR DESCRIPTION
**Pull request by anaik91**
## Description
What's changed, or what was fixed?

- Fixed API redployment
- Fixed API basepath 

## Issues Fixed
- Fixes #

## Housekeeping
(please check all that apply [x], do not edit the text)
- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

## Full Repo Validation Required
(please check all that apply [x], do not edit the text)
- [ ] PR requires full pipeline run (Run for changes only by default).

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 **/node_modules/**
 **/generated/**
-
+target/
+export/
```
Comment (OmidTahouri) [inline .gitignore:4]: could you move these two to a .gitignore in your project's directory (instead of root .gitignore)

**Review by OmidTahouri — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (anaik91) [reply .gitignore]: Done

**Review by OmidTahouri — APPROVED** [review]

**Pull request merged**